cmd/root.go

```diff
@@ -128,9 +128,23 @@ var rootCmd = &cobra.Command{
 						}
 					}
 					if !allowCmdWithJWTV1Store {
-						return fmt.Errorf(`this version of nsc only supports jwtV2. To upgrade the v1 store %#q - type "%s upgrade-jwt". `+
-							`Alternatively you can downgrade %s to a compatible version using: "%s update -version 0.5.0"\n`,
-							store.GetName(), os.Args[0], os.Args[0], os.Args[0])
+						return fmt.Errorf(`this version of nsc only supports jwtV2. 
+If you are using a managed service, check your provider for 
+instructions on how to update your project. In most cases 
+all you need to do is:
+"%s add operator --force -u <url provided by your service>"
+
+If your services is well known, such as Synadia's NGS:
```

Comment:
Should be "If your services are well known"...

Reply:
Maybe stick to singular
If your service is well known